starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,65 @@
 // Names and Input
 
+console.log("I'm Ready!");
+
+var hacker1 = "Celine";
+console.log("The driver's name is " + hacker1);
+
+var hacker2 = "Christina";
+console.log("The navigator's name is " + hacker2);
 
 //Conditionals
 
+if (hacker1.length < hacker2.length){
+  console.log("Yo, navigator got the longest name , it has " + hacker2.length + " characters");
+}
+else if (hacker1.length > hacker2.length){
+  console.log("the Driver has the longest name, it has " + hacker1.length + " characters");
+} else {
+  console.log("wow, you both got equally long names," + hacker1.length + " characters");  
+}
+
+var toUpperCaseString = "";
+for(var i = 0; i <= hacker1.length-1; i++){
+toUpperCaseString += hacker1[i] + " " ;
+}
+console.log(toUpperCaseString.toUpperCase());
+
+var reverseString = "";
+for(var i = hacker2.length-1; i >= 0; i--){
+reverseString += hacker2[i];
+}
+console.log(reverseString);
+
+var names = [hacker2, hacker1];
```

Comment:
your sort function doesn't work as you think it does- 

if I put my name in as 'abi' your names appear first. but if I put 'Abi' my name is first. 

This is because it's sorting by the [code](https://www.w3schools.com/charsets/ref_html_ascii.asp) of the letter. 

you can check out this [link](https://stackoverflow.com/questions/6712034/sort-array-by-firstname-alphabetically-in-javascript) on another way to sort by name. I can take a look with you in person if you'd like!